### ISSUE TITLE
Exception occurring when running jmClean

### ISSUE THREAD

**Issue by embee1981** [jmeter-gradle-plugin/jmeter-gradle-plugin#79]
Version : 1.0.6-2.13

On running jmClean, the build/jmeter-report dir is fully cleaned up however the following exception occurs

```
:jmClean FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jmClean'.
> Cannot invoke method delete() on null object
```

Stack trace snippet

```
Caused by: java.lang.NullPointerException: Cannot invoke method delete() on null object
    at net.foragerr.jmeter.gradle.plugins.TaskJMClean.jmClean(TaskJMClean.groovy:24)
    at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
```

**Comment by foragerr** [comment]
I believe this only happens when report folder is already empty? would you be able to confirm?
It still is a defect, regardless.

**Comment by DKroot** [comment]
I'm having the same issue after upgrade from 1.0.3 to 1.0.6. It does not seem to depend on what's in the report folder. I crated a dummy file there. It was deleted successfully, but the task fails afterwards.

**Comment by DKroot** [comment]
For those facing the same issue, I am able to use 1.0.6 with the following workaround:

```
// Replaces broken jmClean in net.foragerr.jmeter 1.0.6
task cleanPerfResults(type: Delete) {
    delete fileTree("${buildDir}/jmeter-report")
}
```

**Comment by DKroot** [comment]
A better workaround option that gets rid of an extra task by using automatic cleanup (http://mrhaki.blogspot.com/2010/10/gradle-goodness-automatic-clean-tasks.html):

```
apply plugin: 'base'

// cleanPerfTest replaces broken jmClean in net.foragerr.jmeter 1.0.6
task perfTest(dependsOn: ['cleanPerfTest', 'jmRun']) {
    group = 'Verification'
    description = 'Runs (cleanly) performance tests on a deployed application and collects the app log if available.\n' +
        'Use --no-daemon to see progress in the stdout.\n' +
        'Use --info to see all JMeter command-line arguments.'
    //inputs.file srcFile
    outputs.dir "${buildDir}/jmeter-report"
}
```

**Comment by foragerr** [comment]
Fixed in dev build 1.0.7-2.13-SNAPSHOT
